docs/source/conf.py

```diff
@@ -184,6 +201,13 @@
             "icon": "fa-brands fa-mastodon",  # Font Awesome icon
         },
     ],
+    "header_links_before_dropdown": 4,
```

Comment:
Anyway of having 5 and keeping the Gallery visible (not in dropdown)?

Reply:
The issue was that I wanted everything on one line even when the user wasn't in full-screen, which is basically impossible unless the navbar is slimmed down. So I will keep it like it is for now, and then change it later if need be.